tools/testsys/src/wait.rs

```diff
@@ -0,0 +1,352 @@
+use crate::error::{self, Result};
+use futures::future::join_all;
+use log::{error, info, warn};
+use snafu::{OptionExt, ResultExt};
+use std::collections::HashMap;
+use std::time::Duration;
+use testsys_model::clients::{AllowNotFound, CrdClient};
+use testsys_model::test_manager::TestManager;
+use testsys_model::{
+    Crd, CrdExt, DestructionPolicy, Outcome, Resource, ResourceError, TaskState, Test, TestResults,
+};
+
+const WAIT_TIMEOUT_SECONDS: u64 = 600;
+const WAIT_CHECK_SECONDS: u64 = 10;
```

Comment:
Why not just store these as `Duration` to avoid needing to encode the unit in the variable name?